data/Text.csv

```diff
@@ -309,3 +308,48 @@ T00191,Stories Old and New,,en,P0192,G05,PL0002,1958,Renmin wenxue chubanshe,162
 T00191,古今小说,,zh,P0192,G05,PL0002,1958,Renmin wenxue chubanshe,1620,no,,,642,,,2018-01-25,OS,2018-01-25,OS
 T00192,The unjustly murdered woman from the neighbourhood,,en,P0220,G03,,2010,,2010,NULL,http://www.xujuneberlein.com/rem57.html,57,ONLINE SOURCE,,,2018-01-25,OS,2018-01-25,OS
 T00192,被冤杀的街坊大妈,,zh,P0220,G03,,2010,,2010,NULL,http://www.xujuneberlein.com/rem57.html,57,ONLINE SOURCE,,,2018-01-25,OS,2018-01-25,OS
+T00170,The Seventies,The broken pieces in my heart,en,P0221,G03,,,,,,T00044,,235-256,,,2018-02-01,OS,2018-02-01,OS
+T00170,七十年代,我心中的碎片,zh,P0221,G03,,,,,,T00044,,235-256,,,2018-02-01,OS,2018-02-01,OS
+T00171,unknown,,en,P0098,G24,unknown,unknown,unknown,unknown,yes,,,,very likely that for the 1960s books by this author refer to a neibu publication,,2018-02-01,OS,2018-02-01,OS
```

Comment:
these depend on the split of the table so it's ok to merge like this and change later